src/components/Forum/CreateItem.js

```diff
@@ -34,25 +34,30 @@ export function CreateItem(props) {
 			id: uuid()
 		};
 		firebase.database().ref('forum').push(newLink);
+        
+
+		
+
 	}
 	function handleClick() {
 		setIsClicked((value) => !value);
```

Comment:
what does it mean, if you use ```setIsClicked``` then you should use ```isClicked``` somewhere

Reply:
I used it before, then changed my logic and forgot to delete it